lib/event_store_client/deserialized_event.rb

```diff
@@ -6,9 +6,11 @@ module EventStoreClient
   class DeserializedEvent
     InvalidDataError = Class.new(StandardError)
 
+    attr_reader :id
+    attr_reader :type
+    attr_reader :title
     attr_reader :data
     attr_reader :metadata
-    attr_reader :type
```

Comment:
why did you move the type above? I'd understand if you'd order attrs alphabetically, but the data and metadata are last items here, so I'm not sure what was the reason.

Reply:
For me make sense to have id than type than attributes than data than metadata. But also you are also right that makes sense to have ordered alphabetically. So do you think we should keep alphabetical order?